site/src/examples/list-box/Scrolling.tsx

```diff
@@ -0,0 +1,19 @@
+import { Option, ListBox } from "@salt-ds/core";
+import { ReactElement } from "react";
+import { largestCities } from "./exampleData";
+
+export const Scrolling = (): ReactElement => {
+  return (
+    <ListBox
+      style={{
+        maxHeight:
+          "calc((var(--salt-size-base) + var(--salt-spacing-100)) * 8)",
+        width: "30%",
```

Comment:
for site examples, is it better to use multiplier for size token to cater for different viewport?

Reply:
cc @yunjungyeh